SimCalorimetry/EcalSimAlgos/interface/ComponentShape.h

```diff
@@ -0,0 +1,34 @@
+#ifndef EcalSimAlgos_ComponentShape_h
```

Comment:
According to the naming rules this should be
```suggestion
#ifndef SimCalorimetry_EcalSimAlgos_ComponentShape_h
```

Reply:
Changed.